micromasters/urls.py

```diff
@@ -52,6 +53,8 @@
         name='financial_aid_action'),
     url(r'^api/v0/financial_aid_course_price/(?P<program_id>[\d]+)/$',
         GetLearnerPriceForCourseView.as_view(), name='financial_aid_course_price'),
+    url(r'^api/v0/financial_aid_documents_sent/(?P<financial_aid_id>[\d]+)/$',
+        DocumentsSentView.as_view(), name='financial_aid_documents_sent'),
```

Comment:
do you see anything wrong with just changing these names to be more general? `DocumentsSentView` -> `FinancialAidDetailView`, `DocumentsSentSerializer` -> `FinancialAidSerializer`, `api/v0/financial_aid_documents_sent/(?P<financial_aid_id>[\d]+)/` -> `api/v0/financial_aid/(?P<financial_aid_id>[\d]+)/`. one idea is that you could expect the desired status to be passed to the endpoint in the request body in addition to `date_documents_sent`. honestly, if this sounds time-consuming, just say so and we can drop it. it's just a little irksome that the pattern right now is to add a brand new, specifically-named view for every type of action we want to perform via API, and that pattern is going to continue to be followed